src/alpaca_typer.erl

```diff
@@ -4760,19 +4781,7 @@ different_arity_test_() ->
               ?assertMatch(
                  {error, {not_found, _, <<"add">>, 2}},
                  module_typ_and_parse(Code))
-      end
-    , fun() ->
```

Comment:
Yes - the test no longer succeeded as `f x` now becomes a valid currying of `f`, meaning that it would return an error as previously expected. I tested that the currying works correctly elsewhere though. I believe there's also another test which checks than an error is returned when too many arguments are passed. Would it be preferable to keep this test as well and pass too many arguments here, perhaps? Or change the assert to check that it returns without error?